docs/Salesforce-batchsink.md

```diff
@@ -81,6 +81,13 @@ Update - updates existing records based on Id field.
 **Upsert External ID Field:** External id field name. It is used only if operation is upsert.
 The field specified can be either 'Id' or any customly created field, which has external id attribute set.
 
+**Concurrency Mode:** The concurrency mode for the job. The valid values are:<br>
```

Comment:
nit: "The concurrency mode for the job." -> "The concurrency mode for the bulk job."

Reply:
done e130c2d587656a458efc1c7b64df2f795710543c